### PR TITLE
Update the load test dashboard panels to new format

### PR DESCRIPTION
**Pull request by agnivade**

### CODE REVIEW

File: deployment/terraform/assets/dashboard_data.json [hunk 1]
```diff
@@ -1859,5 +2047,6 @@
   "timezone": "browser",
   "title": "Mattermost Load-Test Performance Monitoring",
   "uid": "000000011",
-  "version": 4
+  "version": 3,
```
Comment (agnivade) [inline deployment/terraform/assets/dashboard_data.json:2050]: This got set back to 3. I didn't see any problems with this, but lmk if you guys want me to reset it to 4.

File: deployment/terraform/assets/dashboard_data.json [hunk 2]
```diff
@@ -1790,23 +1975,27 @@
...
   ],
   "templating": {
     "list": [
       {
-        "allValue": null,
         "current": {
-          "text": "",
+          "selected": true,
+          "text": [
+            "192.168.254.1:8067"
```
Comment (streamer45) [inline deployment/terraform/assets/dashboard_data.json:1989]: Not sure if these should be set.

**Review by streamer45 — APPROVED** [review]
Thanks!

File: deployment/terraform/assets/dashboard_data.json (hunk 2 above)
Comment (agnivade) [reply deployment/terraform/assets/dashboard_data.json:1989]: Doesn't need to be. It was just part of the exported JSON file. I can remove them to keep it clean.

**Review by agarciamontoro — APPROVED** [review]
Are there any visible changes in the dashboards? Or is this just purely about the format itself?

**Comment by agnivade** [comment]
No visible changes. Apart from the fact that now the new panels look much better. :sparkles:

**Pull request merged**